apps/etl-func/local.settings.json.example

```diff
@@ -5,8 +5,9 @@
     "FUNCTIONS_WORKER_RUNTIME": "node",
     "MESSAGE_CONTENT_STORAGE_URI": "https://testgiovannist.blob.core.windows.net/",
     "MESSAGE_CONTENT_CONTAINER_NAME": "messages",
-    "EVENTHUB_CONNECTION_URI": "io-d-evh-cdc.servicebus.windows.net",
+    "MESSAGE_EVENTHUB_CONNECTION_URI": "io-d-evh-cdc.servicebus.windows.net",
```

Comment:
If we want to change this variable name than those files should be updated:


apps/etl-func/src/adapters/config.ts
apps/etl-func/src/adapters/env.ts
infra/resources/_modules/web_apps/locals.tf
infra/resources/_modules/web_apps/variables.tf
infra/resources/prod/web_apps.tf

Reply:
We don't need to update the name of this variable because we are going to use the same event hub, ita was a mistake during the merge I believe, nice catch.
[2e3aa71](https://github.com/pagopa/io-messages/pull/63/commits/2e3aa712168c5b40970a4301115db7107e85a896)